CC/src/DistanceComputationTools.cpp

```diff
@@ -2500,9 +2500,9 @@ int DistanceComputationTools::computeCloud2RectangleEquation(GenericIndexedCloud
 	widthXVec = rotationTransform * widthXVec;
 	widthYVec = rotationTransform * widthYVec;
 	normalVector = rotationTransform * normalVector;
-	double planeDistance = CCVector3::vdotd(center.u, normalVector.u);
-	double d = 0.0;
-	double dSumSq = 0.0;
+	PointCoordinateType planeDistance = CCVector3::vdotd(center.u, normalVector.u);
+	PointCoordinateType d = 0.0;
```

Comment:
= 0 (otherwise you'll have a cast warning when PointCoordinateType is float - you can actually compile CC with PC = float or PC = double)